### ISSUE TITLE
Correct assumption for testcase 59 & 60?

### ISSUE THREAD

**Issue by ValerieNexuz** [hl7-be/lab#244]
In testcase 59 - 9A13 and testcase 60 - 9A14 loinc codes (39112-8 & 80264-5) are used that don't have a dutch translation. For the loinc codes without dutch description we want to show the description that is send via the observation. 

In a previous question there was explained that GP softs can chose which description they want to show. So the behaviour we are applying for loinc codes without dutch desciptions is also correct?

**Comment by bdc-ehealth** [comment]
Indeed, this is a correct assumption. The best thing to do in that case (but that is a personal opinion) is showing the label from the file.
